frontend/public/style/_layout.scss

```diff
@@ -6,6 +6,10 @@ body,
 .pf-c-page__main-section,
 .pf-c-drawer {
   height: 100%; // so .co-p-has-sidebar, .yaml-editor are full height
+
+  &__content {
+      z-index: 1;
```

Comment:
@jcaianirh this fix breaks full screen terminal.  Will open a bugzilla.  cc:  @spadgett